src/docfx/config/Config.cs

```diff
@@ -83,6 +83,12 @@ internal sealed class Config
         /// </summary>
         public readonly Dictionary<string, string> Dependencies = new Dictionary<string, string>(PathUtility.PathComparer);
 
+        /// <summary>
+        /// Gets the map from resolve alias to relative path relatived to `docfx.yml` file
+        /// Default path of tilde symbol `~` is '.'
+        /// </summary>
+        public readonly Dictionary<string, string> ResolveAlias = new Dictionary<string, string>(PathUtility.PathComparer);
```

Comment:
nit: can just initialize `ResolveAlias` to `{ '~': '.' }` here so we can see the default value simply by looking at the definition here